prime-angular-frontend/src/app/modules/gis-enrolment/pages/ldap-information-page/ldap-information-page.component.html

```diff
@@ -3,19 +3,19 @@
         (ngSubmit)="onSubmit()"
         novalidate
         autocomplete="off">
-  <app-page-header>{{ title }}</app-page-header>
```

Comment:
I'm assuming this and a few other files are just auto-save from formatting. If you don't have code changes in the files try not to commit formatting like this with PR's takes time to decipher if anything changed as well as if they are not in the direct area that code was changed it an potentially not get tested and bugs can creep in if something goes wrong.

Reply:
I had code changes in all those files, but then reverted I will do that work in another ticket. I will try to remove those from the commit.

Reply:
Reverted the fgiles